### ISSUE TITLE
Expected training time

### ISSUE THREAD

**Issue by SanderGielisse** [junyanz/BicycleGAN#66]
Currently running the train_edges2shoes.sh script on an overclocked GTX 1070. How long did it take for you to finish and on which GPU? At current speed it will take about 15 days for me, not sure if I am doing something wrong here. GPU is active, but nowhere near fully loaded. All defaults settings used as provided by the script.

**Comment by junyanz** [comment]
It takes 2-3 days on a Titan X. GTX 1070 is a little bit slow. You can also check the data loading time. Increasing `--num_threads` might help sometimes. Make sure that your GPU load is close to 100%.

**Comment by SanderGielisse** [comment]
Thank you for the suggestion.

Training became slower and slower over time maxing out a single core. Training speed started at 6300 seconds per epoch at the beginning, but later taking up to 24000 seconds per epoch. Data loading time was low, so I guess that was not the problem.

One thing I did change was the image preview interval which I changed from 400 to 10, so there might be something going wrong there when flooding visdom with too many previews? Just restarted the training session with the interval put back at 400, seeing how that goes. Will get back with more details asap.
